README.adoc

```diff
@@ -418,3 +420,22 @@ In response you should get ID of created item, using that you can retrieve the w
 ----
 $ ./bin/wit-cli show workitem --id <ID> -H localhost:8080 --pp
 ----
+
+== Prod preview setup
+
+In order to setup wit to work against link:https://prod-preview.openshift.io/[prod preview] one needs to update `token.publickey` in config.yaml.
+
+NOTE: Public key can be obtained from https://sso.prod-preview.openshift.io/auth/realms/fabric8/
+
+Also, the following environment variables must be set:
+
+* `export F8_DEVELOPER_MODE_ENABLED=1`
+* `export F8_OPENSHIFT_TENANT_MASTERURL="https://console.free-int.openshift.com/"`
+* `export F8_KEYCLOAK_REALM="fabric8"`
```

Comment:
@ibuziuk make sure that F8_KEYCLOAK_REALM is not set (then fabric8-test realm will be used by default).
Actually make sure you don't set any F8_KEYCLOAK_* to use the default values.
And yes, you need to set F8_DEVELOPER_MODE_ENABLED=1 to make WIT use the default Keycloak URL instead of  trying to convert WIT requests to SSO requests (api.openshift.io -> sso.openshift.io for example). Or you would have to set the KC URL via env var.